### PR TITLE
Update to minecraft version 1.14.4

### PR DESCRIPTION
**Pull request by polo123qwe**
Hello, I've updated the mod to work in the 1.14.4, the only functionality that is missing currently is the evict command.

It would be nice to merge it into a branch of its own (1.14.4)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,77 +1,149 @@
...
-apply plugin: 'net.minecraftforge.gradle.forge'
-//Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
+apply plugin: 'net.minecraftforge.gradle'
+// Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
+apply plugin: 'eclipse'
+apply plugin: 'maven-publish'
 
-
-version = "6"
+version = "7.0.0"
```
Comment (alfuken) [inline build.gradle]: Instead of versions I prefer to use just build numbers for now. :)

File: build.gradle [hunk 2]
```diff
@@ -1,77 +1,149 @@
...
-    }
-        
-    // copy everything else except the mcmod.info
-    from(sourceSets.main.resources.srcDirs) {
-        exclude 'mcmod.info'
+// Example for how to get properties into the manifest for reading by the runtime..
+jar {
+    manifest {
+        attributes([
+            "Specification-Title": "examplemod",
```
Comment (alfuken) [inline build.gradle]: this needs an update

File: src/main/java/lime/chunk_claim/ClaimData.java [hunk 3]
```diff
@@ -117,134 +111,112 @@ static int getClaimsCount(EntityPlayer player)
     private String owner;
     private Set<String> members = new HashSet<>();
 
-    ClaimData()
-    {
+    ClaimData() {
         this.x = Integer.MAX_VALUE;
         this.z = Integer.MAX_VALUE;
-        this.dimension = Integer.MAX_VALUE;
+        this.dimension = DimensionType.OVERWORLD.getId();
```
Comment (alfuken) [inline src/main/java/lime/chunk_claim/ClaimData.java]: This was intentionally set to be something that will never match anything, as ClaimData with no values intended to be a placeholder that just passes all checks. Kinda.

File: src/main/java/lime/chunk_claim/Configuration.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                    .translation("chunkClaim.config.disable_item_activation")
+                    .define("disable_item_activation", true);
+            disable_explosions = builder
+                    .comment("Disable all and any explosion damage to blocks inside claimed chunk.")
+                    .translation("chunkClaim.config.disable_explosions")
+                    .define("disable_explosions", false);
+            max_claimed_chunks = builder
+                    .comment("Disable all and any explosion damage to blocks inside claimed chunk.")
+                    .translation("chunkClaim.config.max_claimed_chunks")
+                    .define("max_claimed_chunks", 9);
```
Comment (alfuken) [inline src/main/java/lime/chunk_claim/Configuration.java]: Configuration sure did become complicated in 1.14...

File: src/main/resources/META-INF/mods.toml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+# A URL to query for updates for this mod. See the JSON update specification <here>
+#updateJSONURL="" #optional
+# A URL for the "homepage" for this mod, displayed in the mod UI
+#displayURL="" #optional
+# A file name (in the root of the mod JAR) containing a logo for display
+#logoFile="" #optional
+# A text field displayed in the mod UI
+credits="The Forge and FML guys, for making this possible" #optional
+# A text field displayed in the mod UI
+authors="Lime" #optional
```
Comment (alfuken) [inline src/main/resources/META-INF/mods.toml]: This needs editing, too ;)

File: build.gradle (hunk 1 above)
Comment (polo123qwe) [reply build.gradle]: The logs of forge complain if there is no semantic versioning though

Comment (alfuken) [reply build.gradle]: hm... how about `1.14.7` then? as in `mc major version, mc minor version, build number`

Comment (alfuken) [reply build.gradle]: or `1.14.4.build_no`

Comment (polo123qwe) [reply build.gradle]: Why don't you have something like 1.X.0. X being the build number, and for 1.14.4 something like 2.X.0

Comment (alfuken) [reply build.gradle]: alright, that fits too

**Pull request merged**